blog/2020-10-23-supervisor-update.md

```diff
@@ -0,0 +1,98 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorTwitter: ludeeus
+title: "Calling all add-on developers!"
+---
+
+## Documentation
+
+First off, our [API documentation][api_docs] has been moved to the developer documentation site, during this move it also got a style update to make it easier to navigate. Some of the endpoints are still missing some content, if you have not yet met your quota for [Hacktoberfest] maybe you want to contribute some more details to our API descriptions?
+
+## API Changes
+
+- Using the `/homeassistant/*` endpoints is deprecated, and will be removed later this year. You need to use `/core/*` instead.
+- Using `http://hassio/` is deprecated, and will be removed later this year. You need to use `http://supervisor/` instead.
+- Using `HASSIO_TOKEN` is deprecated, and will be removed later this year. You need to use `SUPERVISOR_TOKEN` instead.
+- Deleting snapshots with `POST` calling `/supervisor/snapshots/<slug>/remove` is deprecated, and will be removed later this year. You need to use the `DELETE` method when calling `/supervisor/snapshots/<slug>` instead.
+- Using `X-Hassio-Key` header as authentication method is deprecated, and will be removed later this year. You need use an authorization header with a `Bearer` token instead.
+
+The [API documentation][api_docs] have been updated to reflect these changes.
```

Comment:
Looking at the docs, I don't see the old APIs. Shouldn't they be there but marked as deprecated? Or are we going for just the current APIs?

Reply:
If we document old deprecated endpoints, folks will use them.
I think the docs should reflect the current non-deprecated endpoints and methods.

Many of these were "soft" deprecated earlier, but I could not find anywhere that we had communicated that to add-on devs